codebreaker.js

```diff
@@ -1,26 +1,29 @@
 "use strict"
 
+import * as fs from 'fs';
+import * as path from 'path';
+import * as os from 'os';
 import initPrompt from 'prompt-sync';
 
 const prompt = initPrompt();
 
 function allTrue(a, b){
-    return a && b
+    return a && b;
```

Comment:
Don't forget semi-colons at the ends of lines.  While it's not strictly necessary it's considered best practice because it can help prevent tricky-to-debug bugs